ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -83,7 +72,7 @@ def __init__(self, rabbitmq_addr, mongodb_addr, processor_name, ocrd_tool: dict,
         self.processor_name = processor_name
         # The processor class to be used to instantiate the processor
         # Think of this as a func pointer to the constructor of the respective OCR-D processor
-        self.processor_class = processor_class
+        self.ProcessorClass = processor_class
```

Comment:
Purely for readability purposes through the network module - to follow the convention of naming classes.

Reply:
Sure, but it's still an attribute referencing a class. I don't mind going back to `processor_class` since there is `_class` in any way. `ProcessorClass` -> `processor_class`